tests/torch/test_model_graph_manager.py

```diff
@@ -370,6 +371,24 @@ def test_update_fused_bias(model_cls):
         assert torch.all(torch.isclose(model.conv.bias * model.bn.weight + model.bn.bias, ref_new_bias))
 
 
+@pytest.mark.parametrize(
+    "metatype,weight_port_id,ndims,expected_reduction_axes",
+    [
+        (om.PTAtenEmbeddingMetatype, 0, 4, [1]),
+        (om.PTLinearMetatype, 0, 4, [3]),
+        (om.PTMatMulMetatype, 0, 4, [3]),
+        (om.PTMatMulMetatype, 1, 4, [2]),
+        (om.PTAddmmMetatype, 1, 4, [3]),
+        (om.PTAddmmMetatype, 2, 4, [2]),
+        (om.PTMatMulMetatype, 1, 1, [0]),
+        (om.PTMatMulMetatype, 1, 2, [0]),
+    ],
```

Comment:
Test contain only part of metatypes that supports by get_weight_compression_reduction_axes, add all other